src/main/resources/CodyBundle.properties

```diff
@@ -108,3 +108,5 @@ popup.select-chat=Select Chat
 popup.remove-chat=Remove Chat
 popup.remove-all-chats=Remove All Chats
 
+PromptPanel.ask-cody=Message (use @ to include files)...
```

Comment:
@danielmarquespt WDYT?

To me that "Message" alone looks very incomplete.
I would prefer something like fix of old and new:
"Ask a question about this code (you can use @ to include files)..."
Unless that is too wordy.

Reply:
I just tried to make it as similar to VS Code as possible

Reply:
VS Code: 
![image](https://github.com/sourcegraph/jetbrains/assets/19799111/00574ec5-e987-4aaa-a7cf-d6fa2de3b09b)

Reply:
Let's go with:

`Message (type @ to include specific files as context)`

if it is a follow-up, (low priority)

`Follow-up message (type @ to include specific files as context)`